Source/ZenLib/File.cpp

```diff
@@ -269,7 +269,7 @@ bool File::Open (const tstring &File_Name_, access_t Access)
                                            else
                                                 mode=ios_base::binary|ios_base::out|ios_base::app;
                                            break;
-                default                  : ;
+                default                  : mode = ios_base::binary | ios_base::in; break;

```

Comment:
`break` could be removed.
We could also remove the line with `case Access_Read`